StackCalculator/StackCalculatorProj/Stacks/Impl/StackLinkedList.cs

```diff
@@ -0,0 +1,58 @@
+﻿namespace StackCalculatorProj.Stacks.Impl;
+
+/// <summary>
+/// Stack class based on Linked List
+/// </summary>
+public class StackLinkedList <T> : IStack<T>
+{
+    private Node? _top;
+    private int _size = 0;
+    public int Size => _size;
+
+    /// <summary>
+    /// Checks if current <see cref="StackArray{T}"/> instance is empty
+    /// </summary>
+    public bool IsEmpty()
+        => _size == 0;
+    
+    
+    /// <summary>
```

Comment:
```suggestion
        => _size == 0;
    
    /// <summary>
```
Две пустые строки подряд никогда не ставятся